code/game/gamemodes/wizard/raginmages.dm

```diff
@@ -7,6 +7,7 @@
 	var/making_mage = 0
 	var/mages_made = 1
 	var/time_checked = 0
+	var/bullshit_mode = 0 // requested by hornygranny
 
 /datum/game_mode/wizard/announce()
 	world << "<B>The current game mode is - Ragin' Mages!</B>"
```

Comment:
Is it intentional that the crew doesn't get to know what hellacious fuck-you is imminent?